lib/xcode/install/install.rb

```diff
@@ -18,6 +18,7 @@ def self.options
          ['--no-progress', 'Don’t show download progress.'],
          ['--no-clean', 'Don’t delete DMG after installation.'],
          ['--no-show-release-notes', 'Don’t open release notes in browser after installation.'],
+         ['--cache-url-base', 'Base URL containing the Xcode DMG (e.g. "http://10.1.1.1/XcodeCache". Overrides --url unless a failure occurs.'],
```

Comment:
This seems to be the "parent" (or containing folder) of what would be the `--url`. How about keeping them close to each other (in the code), and using a more similar name, such as `--url-base`? I'd mention "cache" in the param description for both `--url` and `--url-base` - that would help a lot IMO 😊 

_I would probably want to rename `--url` to `--cache-url` but that would be a breaking change 🙈 so let's leave it as is for now at least, and just improve the documentation? 😇_